org.eclipse.ice.bats/pom.xml

```diff
@@ -0,0 +1,114 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.eclipse.ice</groupId>
+    <artifactId>org.eclipse.ice.bats</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>Eclipse ICE BATS</name>
+    <description>Basic Artifact Tracking System</description>
+
+    <properties>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Can we bump to 11 from 8? The rest of ICE is now on 11.

Reply:
Done and tested!